scripts/create-package.ts

```diff
@@ -295,7 +292,6 @@ function validatePackageName(packageName: string) {
  * @param componentName - Component name.
  */
 function addPackageToUi(packageName: string, directory: string, componentName?: string) {
-  //exec(`yarn lerna add ${packageName} --exact --scope=@littlespoon/ui`)
   const packagePath = resolve(path.packages, directory)
   exec(`yarn workspace @littlespoon/ui add file:${packagePath} --exact`)
```

Comment:
Can you explain the difference with the commented-out code?

Reply:
As you can see in the commented out code, the Lerna `add` command was executed to install/add a dependency to a internal package (`@littlespoon/ui`). Recently, the `add` command was removed from Lerna CLI. Then, in order to achieve what I just described, we now need to use the yarn `add` command

Reply:
Got it, thanks!